### DISCUSSION TITLE
[Showcase] My Site "Robotdazero.it"

### DISCUSSION THREAD

**Discussion by sebadima** [gethyas/hyas#453] [Showcase]
### Site name

Robotdazero.it

### Public URL

https://www.robotdazero.it/

### Short description

Robotdazero is an italian site about  Arduino, ESP32 and Internet of Things enthusiasts in Italy.

### Full description

 The site offers guides, tutorials, insights and updates on all the news related to these topics.

The Robotdazero  blog is a reference point for those who want to stay updated on the latest news and trends in the world of Arduino and IoT. The blog hosts in-depth articles, tutorials, reviews and interviews with industry experts.

Guides and tutorials for beginners and experts,
Insights on specific topics, such as programming, electronics, home automation, robotics and IoT,
Updates on the latest Arduino, ESP32 and IoT news.

### Terms

- [X] I agree to the [Terms of Service](https://gethyas.com/terms/).

**Comment by h-enk** [comment]
Thanks for sharing — interesting content and very professionally done!

https://getdoks.org/showcase/
